site/content/en/docs/Guides/feature-stages.md

```diff
@@ -30,8 +30,8 @@ The current set of `alpha` and `beta` feature gates:
 |-----------------------------------------------------------------------------------------------------------------------------|------------------------------------|----------|---------|--------|
 | [CountsAndLists](https://github.com/googleforgames/agones/issues/2716)                                                      | `CountsAndLists`                   | Enabled  | `Beta`  | 1.41.0 |
 | [DisableResyncOnSDKServer](https://github.com/googleforgames/agones/issues/3377)                                            | `DisableResyncOnSDKServer`         | Enabled  | `Beta`  | 1.40.0 |
+| [Support for Extended Duration Pods on GKE Autopilot](https://github.com/googleforgames/agones/issues/3386)  | `GKEAutopilotExtendedDurationPods` | Enabled | `Beta` | 1.44.0 |
```

Comment:
This need to be behind a `{{% feature publishVersion="1.44.0" %}}` block since it's only in beta after 1.44 release. Take a look at https://github.com/googleforgames/agones/pull/3916/files#diff-6a76655b1e397b2f3b9955ebe19b8263aa78cd04aed336f6a737d9e408eac30c for details.

Reply:
nit changes with the reference of #3916 .